docs/SNIPPET.txt

```diff
@@ -0,0 +1,8 @@
+module HelloWorld where
+
+import Prelude
+import Data.Maybe (Maybe(Just, Nothing))
```

Comment:
I would replace this with `import Data.Maybe (Maybe(..))`

Otherwise, looks good to me!